packages/vscode-extension/WHATISNEW.md

```diff
@@ -1,22 +1,41 @@
+- [3.0.0 - Nov 29 2021](#300---nov-29-2021)
 - [2.10.0 - Nov 15 2021](#2100---nov-15-2021)
 - [2.9.0 - Nov 01 2021](#290---nov-01-2021)
 - [2.8.0 - Oct 18 2021](#280---oct-18-2021)
 - [2.7.0 - Sep 17 2021](#270---sep-17-2021)
 
-## 2.10.0 - Nov 15, 2021 
+## 3.0.0 - Nov 29, 2021
 
-Incremental version for Teams Toolkit with multiple bugs fixes and the following updates:
+Big version for Teams Toolkit with new features to support customize cloud resources, multiple cloud environments, collaborations and some bug fix.
 
 New Features:
-- Enable developers with the capability to extend Teams apps to run across Microsoft 365. Developers can invoke two important commands from VS Code Command Palette to upgrade their Teams app to Microsoft 365 custom app. Get detail instructions from our [documentation](https://aka.ms/teamsfx-extend-m365). 
 
-  ![Alt Text](https://raw.githubusercontent.com/OfficeDev/TeamsFx/main/packages/vscode-extension/img/teamsfx-extend-m365.png)
-  
-- Provide Teams sample apps which can run across Microsoft 365. Users can get a initial experience of running Teams apps in Microsoft 365 by playing with samples first. Developers can get the samples from sample gallery, or click `Create a new Teams app` -> `Start from a sample`, and select sample apps for Microsoft 365.
+- Adopt ARM templates to provision Azure cloud resources, support customization of cloud resources. Refer to [Provision cloud resources](aka.ms/provision-doc) for more information.
+- Developers can create and manage multiple cloud environments with different customizations for each environment. Refer to [Manage multiple environment](aka.ms/multi-env-doc) for more information.
+
+    ![create multiple environments](./img/create-env.png)
```

Comment:
Use absolute URL like https://raw.githubusercontent.com/OfficeDev/TeamsFx/main/packages/vscode-extension/img/create-env.gif instead of relative file path, as the `img` folder isn't packed into the vsix package.
![image](https://user-images.githubusercontent.com/15644078/143372004-a99e79fb-905f-4019-be78-5b5cee5e8f14.png)

Reply:
fixed.